examples/xt.lua

```diff
@@ -0,0 +1,51 @@
+local xt = require("xtable")
```

Comment:
add your copyright and use a more descriptive name for this script.. what is it doing?

Reply:
*all* your copyright notices, I meant ;-)